### ISSUE TITLE
Error: Package path is not valid. Valid system image paths are:ository...        null.

### ISSUE THREAD

**Issue by karelgo** [NativeScript/NativeScript#5667]
Dear friend, we, the rest of the NativeScript community really
appreciate your feedback! While we are doing all we can to take care of every
issue, sometimes we get overwhelmed. Because of that, we will consider issues
that are not constructive or problems that cannot be reproduced "dead".
Additionally, we will treat feature requests or bug reports with unanswered
questions regarding the behavior/reproduction for more than 20 days "dead". All
"dead" issues will get closed.

## Please, provide the details below:

I am a relatively new mac user os any help would be appreciated.

After having installed nativescript I am trying to create an emulator. I have used the command:

avdmanager create avd --package "android-25" --name "foo"

and the error I get is:

Error: Package path is not valid. Valid system image paths are:ository...       
null.

I can't seem to locate a fix to this problem. Any help?

### Did you verify this is a real problem by searching the [NativeScript Forum](http://forum.nativescript.org) and the [other open issues in this repo](https://github.com/NativeScript/nativescript/issues)?

Yes.

### Tell us about the problem
Please, ensure your title is less than 63 characters long and starts with a capital
letter.

Check the description earlier.

### Which platform(s) does your issue occur on?
iOS/Android/Both

Both.

### Please provide the following version numbers that your issue occurs with:
- CLI: (run `tns --version` to fetch it)
- Cross-platform modules: (check the 'version' attribute in the
`node_modules/tns-core-modules/package.json` file in your project)
- Runtime(s): (look for the `"tns-android"` and `"tns-ios"` properties in the
`package.json` file of your project)
- Plugin(s): (look for the version number in the `package.json` file of your
project)

tns --version
4.0.0
tns doctor
✔ Getting environment information 

✔ Your ANDROID_HOME environment variable is set and points to correct directory.
✔ Your adb from the Android SDK is correctly installed.
✔ The Android SDK is installed.
✔ A compatible Android SDK for compilation is found.
✔ Javac is installed and is configured properly.
✔ The Java Development Kit (JDK) is installed and is configured properly.
✔ Xcode is installed and is configured properly.
✔ xcodeproj is installed and is configured properly.
✔ CocoaPods are installed.
✔ CocoaPods update is not required.
✔ CocoaPods are configured properly.
✔ Your current CocoaPods version is newer than 1.0.0.
✔ Python installed and configured correctly.
✔ The Python 'six' package is found.
No issues were detected.
✔ Getting NativeScript components versions information...
✔ Component nativescript has 4.0.0 version and is up to date.
✖ Component tns-core-modules is not installed.
✖ Component tns-android is not installed.
✖ Component tns-ios is not installed.

### Please tell us how to recreate the issue in as much detail as possible.
The best approach would be to get your code running in the [NativeScript Playground](https://play.nativescript.org) and share the link with us, along with any additional details or steps to reproduce needed for examining the issue there. 

It's a command line issue so I am guessing it has something to do with my file system.
Basically when I run: 
avdmanager create avd --package "android-25" --name "foo". 

The system gives me the error: 
Error: Package path is not valid. Valid system image paths are:ository...       
null.

### Is there code involved? If so, please share the minimal amount of code needed to recreate the problem.

this doesn't work:
avdmanager create avd --package "android-25" --name "foo"

this will also not work:
avdmanager create avd -n test -k "android-25" -c 1000M -f -p ~/.android/avd

**Comment by Juliusdz** [comment]
#5690 You can refer this as it seems I **kinda** solved my problems

**Comment by Juliusdz** [comment]
#5690 You can refer this, as it seems I **kinda** solved my issues. Give a comment if anything goes wrong -  maybe I missed something (I spend all day long installing everything) :)

**Comment by karelgo** [comment]
Thanks for the reply but the actual error that I am having trouble with is:

When I type:

avdmanager create avd --package "android-25" --name "foo"

I get this error:

Error: Package path is not valid. Valid system image paths are:ository...       
null.

The 'are:ository... null' is actually something that my mac returns on the command line =/.

**Comment by Juliusdz** [comment]
Yes, I had exact same error. You need to download Android emulator image / package. So I made all steps as in my [comment](https://github.com/NativeScript/NativeScript/issues/5690#issuecomment-381722271) and now it works fine
